src/NLogShipper/NLogShipper.csproj

```diff
@@ -17,9 +17,10 @@
   </PropertyGroup>
  
   <ItemGroup>
+    <PackageReference Include="log4net" Version="2.0.12" />
```

Comment:
Please remove `log4net` since it should not be a dependency for the NLog-project.